scorer/verifications/aura.py

```diff
@@ -4,18 +4,18 @@
 import time
 
 client = ArangoClient(hosts=config.ARANGO_SERVER)
-system = client.db('_system')
-snapshot = client.db('snapshot')
+system = client.db("_system")
+snapshot = client.db("snapshot")
```

Comment:
This python code base mostly uses single-quotes.  Maybe we need to put a `.editorconfig` file [like this one I created for Aura Backend](https://github.com/Meta-Node/aura-backend/blob/master/.editorconfig).  I'm pretty sure vscode will read `.editorconfig`

Reply:
I like using a standard formatter (in this case ```black```) to format the code automatically whenever I save the file. By default ```black``` replaces all double quotes with single quotes. And I guess we can change that in ```.editorconfig```. or remove ```black``` altogether if you don't like it.

Reply:
Not everyone uses vscode.  I'm not the only one who has contributed to this repo over the years.  I think we should keep with the formatting we've been using for years unless there's something wrong with it.  (There isn't.  It's all compatible with PEP-8.)

Reply:
> By default `black` replaces all double quotes with single quotes.

I think you mean the opposite, right? It prefers double quotes over single quotes?

If we could create an `.editorconfig` (or something similar--like "black" [though I've never used it]) that can match the styles we're already using, that would be perfect.

Actually, that would be good to merge back into the main BrightID Node repo.  Whoever started on the python code originally never created a style config file for editors.  It's nice to have.